### ISSUE TITLE
Multi file upload does not work on div and button elements

### ISSUE THREAD

**Issue by Jarlotee** [danialfarid/angular-file-upload#601]
The if statements below always return undefined because attr is a function not an array:

`if (elem.attr['multiple']) fileElem.attr('multiple', elem.attr['multiple']);`
`if (elem.attr['accept']) fileElem.attr('accept', elem.attr['accept']);`
`if (elem.attr['capture']) fileElem.attr('capture', elem.attr['capture']);`

Fix:

`if (elem.attr('multiple')) fileElem.attr('multiple', elem.attr('multiple'));`
`if (elem.attr('accept')) fileElem.attr('accept', elem.attr('accept'));`
`if (elem.attr('capture')) fileElem.attr('capture', elem.attr('capture'));`

TLDR;  jquery.attr() not jquery.attr[]

**Comment by danialfarid** [comment]
What version are you referring too?

**Comment by Jarlotee** [comment]
3.1.1

**Comment by danialfarid** [comment]
Fixed in version 3.1.2
